### ISSUE TITLE
Please add compiling instructions under README.MD

### ISSUE THREAD

**Issue by longboarder241** [gavinbenda/platinum-md#150]
since there was barely any info on how to build I assumed it was node js due to the addition of electron however doing an npm install bodes a ton of error messages about depreciated packages on both x86_64 and i686 architecture and thus leads to the failure to properly build.  It'd be immensively helpful to add a section on building in that regard.

**Comment by Shadester** [comment]
https://github.com/gavinbenda/platinum-md/blob/master/testing.md

**Comment by longboarder241** [comment]
> https://github.com/gavinbenda/platinum-md/blob/master/testing.md

doing that results in this error;
thomas-pc:[thomash]:~/git/platinum-md$ git checkout rh1-upload
error: pathspec 'rh1-upload' did not match any file(s) known to git

**Comment by Shadester** [comment]
I would just use the master branch.

**Comment by longboarder241** [comment]
> I would just use the master branch.

how would that look as a command?

**Comment by Shadester** [comment]
```git checkout master```

**Comment by gavinbenda** [comment]
Yeah that README was more for development of backend features when working with the RH1.

I'll add a development README shortly, but it's actually quite straight forward!
You need to have Yarn/NodeJS installed, and this is tested with Node 14
There might be some depreciation warnings, but it should still build.

```
> git clone https://github.com/gavinbenda/platinum-md
> cd platinum-md
> yarn
> yarn run dev
```

**Comment by longboarder241** [comment]
> Yeah that README was more for development of backend features when working with the RH1.
> 
> I'll add a development README shortly, but it's actually quite straight forward! You need to have Yarn/NodeJS installed, and this is tested with Node 14 There might be some depreciation warnings, but it should still build.
> 
> ```
> > git clone https://github.com/gavinbenda/platinum-md
> > cd platinum-md
> > yarn
> > yarn run dev
> ```

Thanks! I really appreciate you adding the instructions as itd be really cool to see this adapted to even small devices like the Vaio P one day. Right now I'm compiling under a inter t2060. If I run into any issues I'll definitely let you know.  Keep up the good work Gavin!
